Automattic/jetpack
# Jetpack Social: Add sidebar panel for Social Image Generator

This change copies over the existing SIG panel into the Jetpack Social panel. AND also sets up the post-meta usage.

## Proposed changes:

* Added new panel
* Created a hook which is used to edit and read the post meta for the SIG options.
* Templates are not part of this change.

### Other information:


## Jetpack product discussion



1203750066198187-as-1203857426851217

## Does this pull request change what data or activity we track or use?



no.

## Testing instructions:






* Go to the Social editor tab, you should not see the SIG panel there.
* Add the `publicize-social-image-generator` blog sticker, then go to the Social editor tab, you should see a new panel there called Social Image Generator
* Disabling/Enabling should work. Custom text can be set and saved. Image type is saved.
* Select custom image. You should see the same image picker as with Social, and it should work and save to meta.
* ~~Until #28773 is not merge, you need to rebase this on that to be able to test it.~~
